cloudweatherreport/templates/base.html

```diff
@@ -7,7 +7,7 @@
         <link rel="stylesheet" type="text/css" media="screen" href="https://assets.ubuntu.com/v1/vanilla-framework-version-0.0.65.min.css">
         <!-- link rel="stylesheet" type="text/css" media="all"
               href="http://assets.ubuntu.com/sites/guidelines/css/latest/ubuntu-styles.css"/ -->
-        <link rel="stylesheet" type="text/css" href="http://status.juju.solutions/static/css/base.css" >
+        <link rel="stylesheet" type="text/css" href="http://data.vapour.ws/assets/cwr/css/base.css" >
```

Comment:
I agree with relative link. 
Right now, status.juju.solutions down but the server http://8.19.32.215:8000/recent is up. I think Marco manages the subdomain. The quickest solution right now is server the file from the S3.